### DISCUSSION TITLE
Incorrect WebSocket status code 1005 description, missing status code 1006

### DISCUSSION THREAD

**Discussion by sankirta** [encode/starlette#1577] [Potential Issue] [answered]
It seems that there is a mistake in starlette's WebSocket [status code list](https://github.com/encode/starlette/blob/0.19.0/starlette/status.py#L83).

Status code 1005 has the name `1005_ABNORMAL_CLOSURE` and 1006 is missing altogether. However, [Mozilla MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/CloseEvent/code) indicate that 1005 actually means "no status received". It is status code 1006 that means "abnormal closure".

It looks like what should have been
```
WS_1005_NO_STATUS_RECEIVED = 1005
WS_1006_ABNORMAL_CLOSURE = 1006
```
got mashed together and became just one line:
```
WS_1006_NO_STATUS_RECEIVED = 1005
```

Making starlette backwards-incompatible over this wouldn't make much sense, but it seems reasonable to at least add the two correct lines and document `WS_1005_ABNORMAL_CLOSURE` as deprecated/wrong.

**Comment by Kludex** [comment, marked as answer]
No, what looks like is that we used 1004, when it's reserved, and two constants are wrong.

**Reply by Kludex** [reply]
This was fixed around the time I wrote the message above - just for the record.
